src/templates/singleBusinessPage.js

```diff
@@ -65,6 +69,23 @@ const SingleBusinessPage = ({ data, pageContext }) => {
     zip,
   } = apiResponse || {};
 
+  const shareClicked = event => {
+    if (event && typeof event.preventDefault === 'function')
+      event.preventDefault();
+
+    debugger;
```

Comment:
won't this pause the program?

Reply:
Yikes! Yes that needs to go.  Good catch!